lib/expired_record_cleaner.rb

```diff
@@ -0,0 +1,53 @@
+class ExpiredRecordCleaner
+  attr_reader :verbose, :fiscal_year_start
+
+  def initialize(dtim, ok_to_act = false, verbose = false)
+    @fiscal_year_start = self.class.fiscal_year_start(dtim)
+    @ok_to_act = ok_to_act
```

Comment:
smart -- in what case would we want this to be `false` ? Or is this just a safeguard? 

also: thoughts on named args?